tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ProcessStart/ProcessStartTags.cs

```diff
@@ -0,0 +1,18 @@
+// <copyright file="ProcessStartTags.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.SourceGenerators;
+
+namespace Datadog.Trace.Tagging
+{
+    internal partial class ProcessCommandStartTags : InstrumentationTags
+    {
+        [Tag(Trace.Tags.SpanKind)]
+        public override string SpanKind => SpanKinds.Internal;
+
+        [Tag(Trace.Tags.ProcessEnvironmentVariables)]
+        public string EnviromentVars { get; set; }
```

Comment:
nit: I would prefer naming this property `EnvironmentVariables`, but it's not super important 🤷🏼

Reply:
I agree.